student-cvs/diirfv-7677/cv.html

```diff
@@ -4,17 +4,19 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="../styles.css">
+    <link rel="stylesheet" href="styles.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/cc6144716b.js" crossorigin="anonymous"></script>
     <title>Indira Franchi</title>
 </head>
 
 <body>
     <header>
         <nav class="header-nav">
             <ul>
-                <li><a href="../index.html">INICIO</a></li>
-                <li><a href="./projects.html">PROYECTOS</a></li>
+                <li><a href="/">Inicio</a></li>
```

Comment:
Capitalize > Uppercase.

Reply:
Yes, in my first commit I wrote with capitalize but after a while, I decided to use uppercase